code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -1063,6 +1114,15 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 			pref.b_ears2 = hex2num(copytext(new_earc2, 6, 8))
 			return TOPIC_REFRESH_UPDATE_PREVIEW
 
+	else if(href_list["ear_color3"])
+		var/new_earc3 = input(user, "Choose your character's tertiary ear colour:", "Character Preference",
+			rgb(pref.r_ears3, pref.g_ears3, pref.b_ears3)) as color|null
+		if(new_earc3)
+			pref.r_ears3 = hex2num(copytext(new_earc3, 2, 4))
+			pref.g_ears3 = hex2num(copytext(new_earc3, 4, 6))
+			pref.b_ears3 = hex2num(copytext(new_earc3, 6, 8))

```

Comment:
Similar to above, the various colour combination procs could probably stand to be genericized at some point. They're pretty boilerplate-y, especially since unless Polaris/Virgo are doing something very different to Bay with the component colours you can get away with turning all the `(r|g|b)_(skin|ears|eyes|etc)` vars into hex strings. Not in scope for this PR, but worth considering I think.